.circleci/config.yml

```diff
@@ -49,22 +55,75 @@ jobs:
           command: |
             cd src/test/kotlin
             
-            # Find all Kotlin files that contain tests, write the filenames to filenames.txt and tell CircleCI to split them by filesize across the parallel executors
-            circleci tests glob "**/*.kt" \
-            | xargs grep -l -E '@Test|@ParameterizedTest' \
-            | cut -c 1- \
-            | circleci tests run --command=">filenames.txt xargs echo" \
-            --verbose \
-            --split-by=filesize
+            # Find all packages            
+            all_packages=($(find uk/gov/justice/digital/hmpps/prisonerfromnomismigration -mindepth 1 -maxdepth 1 -type d -exec basename {} \;))
```

Comment:
This whole script isn't the easiest to follow. Not much ❤️ for bash in my mind. So I might move all the code into functions to make it easier to read. Next PR.

Reply:
alternative to the `find` command would be something like
```
ls -p uk/gov/justice/digital/hmpps/prisonerfromnomismigration | grep '/' | tr -d '/'
```
since `ls -p` outputs a trailing `/` for directory items

Reply:
Thanks. If you have any other let me know and I'll implement when I move the code to functions.